composer.json

```diff
@@ -1,8 +1,8 @@
 {
-    "name": "friendsofcake/cakepdf",
+    "name": "kongka/cakepdf",
```

Comment:
Those things should be reverted though then, or make a fresh branch with only relevant changes.